resources/views/home.blade.php

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        @vite(['resources/css/app.css', 'resources/js/app.js'])
+        <title> Blog </title>
+
+    </head>
+    <body class="antialiased">
+        <div style=" width: 900px;" class="max-w-full mx-auto pt-4">
+
+            @if(Session::has('message'))
+                <p class="{{Session::get('color')}} text-green-500">
+                    {{ Session::get('message') }}
+                </p>
+            @endif
+
+            @if(\Illuminate\Support\Facades\Auth::user())
+                <form method="POST" action="/post/create/{{\Illuminate\Support\Facades\Auth::user()->id}}"
+                        enctype="multipart/form-data">
```

Comment:
we are only showing the form to an authenticated user, therefore that comment I made about getting the user_id is `$request->user()->id` is relevant.